bridge/engine/ActiveMediaList.cpp

```diff
@@ -307,6 +309,19 @@ size_t ActiveMediaList::rankSpeakers(float& currentDominantSpeakerScore)
     return speakerCount;
 }
 
+void ActiveMediaList::onNewPtt(const size_t endpointIdHash, utils::Optional<bool> is_ptt)
```

Comment:
Why would we call this method if we do not have an ext id?
Why make it Optional type

Reply:
Agree. I was think like "it does not metter where we check Optional, so why not to defer it to the very end". But we can obviously save time on not calling API at all if the header is not present.